recipes/base_config.rb

```diff
@@ -58,3 +58,8 @@
   # Mount FSx
   include_recipe 'aws-parallelcluster::fsx_mount'
 end
+
+# Install EFA
+if node['cfncluster']['enable_efa'] == 'compute' && node['cfncluster']['cfn_node_type'] == 'ComputeFleet'
+  include_recipe "aws-parallelcluster::_efa_install"
```

Comment:
where are the checks for OS, China and instance type?

Reply:
they're in the frontend

Reply:
Ok, I've revised https://github.com/aws/aws-parallelcluster/pull/1081 so that it enforces the instance type check even when config_sanity check is disabled. This means we can rely on the enable_efa flag to signify that the instance_type and os are supported. Only check needed is to check `enable_efa  = compute` and  `cfn_node_type = ComputeFleet`.